From 5c9bb46db6c2e6085c64785e29b0fe1faf77605b Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Sat, 17 Jun 2017 16:29:13 +0800
Subject: [PATCH 152/741] fmd: use ptp-timer for Fman RTC node name

commit 59791a855edb9a1ea09aa3f1033de923150c47cc from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Layerscape DPAA platforms have updated dts to use ptp-timer
instead of rtc for Fman RTC node name. This patch is to
update it in driver.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c   | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
index a475389b94bd..31f654b4c617 100755
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
@@ -684,9 +684,9 @@ static t_LnxWrpFmDev * ReadFmDevTreeNode (struct platform_device *of_dev)
 
     /* Get the RTC base address and size */
     memset(ids, 0, sizeof(ids));
-    if (WARN_ON(strlen("rtc") >= sizeof(ids[0].name)))
+    if (WARN_ON(strlen("ptp-timer") >= sizeof(ids[0].name)))
         return NULL;
-    strcpy(ids[0].name, "rtc");
+    strcpy(ids[0].name, "ptp-timer");
     if (WARN_ON(strlen("fsl,fman-rtc") >= sizeof(ids[0].compatible)))
         return NULL;
     strcpy(ids[0].compatible, "fsl,fman-rtc");
-- 
2.17.1

